content/katas/fibonacci-sequence/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Fibonacci sequence
+pre: "<b>1. </b>"
+weight: 5
+---
+
+
+ **What is fibonacci sequence?**
+
+> Fibonacci Sequence: is the series of numbers where the next number is found by adding up the 2 numbers before it.
+> in this case n = 2:
+> sequence: 0, 1, 1, 2, 3, 5, 8, 13, 21, 34, ...
+
+**1.** 
+Write a function, fibonacci(n) that takes in a number/int argument (n) and returns the fibonacci sequence from the 1st term of the sequence till the nth term of the sequence. 
```

Comment:
please use Grave accent "``" for code, put your fibonacci(n) inside two grave accents.
like so:  `code` 

Rephrase: Write a function, `fibonacci(n)` that takes in a number/int argument (n) and returns the first nth terms of the sequence.